janus_server/src/client.rs

```diff
@@ -0,0 +1,92 @@
+//! PPM protocol client
```

Comment:
Seeing a client module under `janus_server`, I think we should reorganize this repository into multiple crates in a Cargo workspace. I'll open up an issue with a proposed layout separately.

Reply:
`warp::test` is a good find, thanks! I'm going to try to use it to refactor the test I added into a unit test.